examples/adjoints/adjoints_ex7/initial.C

```diff
@@ -0,0 +1,51 @@
+// The libMesh Finite Element Library.
+// Copyright (C) 2002-2020 Benjamin S. Kirk, John W. Peterson, Roy H. Stogner
+
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+
+
+
+#include "initial.h"
+
+using namespace libMesh;
+
+void read_initial_parameters()
+{
+}
+
+void finish_initialization()
+{
+}
```

Comment:
These too.  Having a single main() that called differing functions in differing codes made some sense at the time.  But once main() is no longer general we don't gain anything by spending dozens of lines of code pretending otherwise.

Reply:
Done.